TS.fsx

```diff
@@ -784,6 +784,7 @@ module Emit =
         for e in tagNameToEleName do
             if iNameToIDependList.ContainsKey e.Value && Seq.contains "HTMLElement" iNameToIDependList.[e.Value] then
                 Pt.Printl "\"%s\": %s;" (e.Key.ToLower()) e.Value
+        Pt.Printl "[index: string]: HTMLElement;"
```

Comment:
i did not add these originally to make the signature help more informative. with this change you will see something like:
`getElementsByTagName<K>(tagname: K)...` instead of `getElementsByTagName<"a" | "div" | ...>(tagname: "a" | "div" | ..)...`